File: AppNotes/AN24_04.py
# AN24_04 -- Accessing Calibration Data

import sys, os
sys.path.append("../")
import Class.TinyRad as TinyRad

# (1) Connect to DemoRad: Check if Brd exists: Problem with USB driver
# (2) Read Calibration Data
# (3) Read Calibration Information

#--------------------------------------------------------------------------
# Setup Connection
#--------------------------------------------------------------------------
Brd = TinyRad.TinyRad('Usb')

#--------------------------------------------------------------------------
# Software Version
#--------------------------------------------------------------------------
Brd.BrdDispSwVers()

#--------------------------------------------------------------------------
# Read position of antennas
TxPosn = Brd.Get('TxPosn')
RxPosn = Brd.Get('RxPosn')

#--------------------------------------------------------------------------
# Receive the calibration information
#--------------------------------------------------------------------------
CalDat          =   Brd.BrdGetCalDat()

print("CalDat: ", CalDat)

#Brd.BrdSetCalDat(CalDat)

File: source_docs/Unusable_Files/MATLAB/TinyRad/AppNotes/AN24_04.m
% AN24_04 -- Accessing Calibration Data 
clear;
close all;

% (1) Connect to DemoRad: Check if Brd exists: Problem with USB driver
% (2) Read Calibration Data
% (3) Read Calibration Information

%--------------------------------------------------------------------------
% Include all necessary directories
%--------------------------------------------------------------------------
CurPath = pwd();
addpath([CurPath,'/../../DemoRadUsb']);
addpath([CurPath,'/../../Class']);

%--------------------------------------------------------------------------
% Setup Connection
%--------------------------------------------------------------------------
Brd         =   TinyRad();

%--------------------------------------------------------------------------
% Software Version
%--------------------------------------------------------------------------
Brd.BrdDispSwVers();

%--------------------------------------------------------------------------
% Read the positions of the transmit and receive antennas
%--------------------------------------------------------------------------
TxPosn          =   Brd.RfGet('TxPosn');
RxPosn          =   Brd.RfGet('RxPosn');

%--------------------------------------------------------------------------
% Display the calibration information
%--------------------------------------------------------------------------
%Brd.BrdDispCalInf();

CalDat          =   Brd.BrdGetCalDat()

CalInf          =   Brd.BrdGetCalInf()
